### DISCUSSION TITLE
Traffic Data Analysis for Modeling and Prediction of Traffic Scenarios (Project 222)

### DISCUSSION THREAD

**Discussion by robertogl** [mathworks/MATLAB-Simulink-Challenge-Project-Hub#54] [Collaborate]
Contribute to the discussion by asking and/or answering questions, commenting, or sharing your ideas for solutions to [project 222](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Traffic%20Data%20Analysis%20for%20Modelling%20and%20Prediction%20of%20Traffic%20Scenarios)

---
*:robot: An AI bot may reply to your comments with suggestions and guidance. It will not solve the project for you. Your comments may be processed by a third-party LLM. To direct a question to a specific person, tag them with @username.*

**Comment by GirolamoOddo** [comment]
Hi, I am a student at University of Modena and interested in working and collaborating on this project

**Comment by Jack0319** [comment]


**Reply by robertogl** [reply]
Hi @Jack0319,

That function is inside the [Utils ](https://github.com/mathworks/MathWorks-Excellence-in-Innovation/tree/main/projects/Traffic%20Data%20Analysis%20for%20Modelling%20and%20Prediction%20of%20Traffic%20Scenarios/Utils) folder, just add it to your path. Please let me know if it does not work.

**Reply by robertogl** [reply]
You are welcome!

**Comment by Santhosh040602** [comment]
Hi, I am a student at Anna University and interested in working and collaborating on this project

**Comment by BarbarosTeoman** [comment]
Hi @robertogl, I have submitted our solution 2.5 weeks ago, but got no feedback at all. The auto response mail was saying "We are going to review your submission and get back to you within the next few days.".

**Reply by robertogl** [reply]
Just done.
